### DISCUSSION TITLE
What does the edge.html created by @next/bundle-analyzer represent?

### DISCUSSION THREAD

**Discussion by Timonwa** [vercel/next.js#44064] [Help]
Hi, good day. Please, I am confused. I am using the @next/bundle-analyzer on my next app. On the [npm page](https://www.npmjs.com/package/@next/bundle-analyzer#:~:text=When%20enabled%20two%20HTML%20files%20(client.html%20and%20server.html)%20will%20be%20outputted%20to%20%3CdistDir%3E/analyze/.%20One%20will%20be%20for%20the%20server%20bundle%2C%20one%20for%20the%20browser%20bundle.), it says that 

```
When enabled, two HTML files (client.html and server.html) will be outputted to <distDir>/analyze/. 

One will be for the server bundle and one for the browser bundle.
```

But it outputs `client.html, nodejs.html, and edge.html` instead when I run it. I deduce that the `client.html` is for the `client side`, and the `nodejs.html` is for the `server side`. But what does the `edge.html` represent? Because when I view the page in my browser, it does not give me any data. 

![ice_screenshot_20221215-201103](https://user-images.githubusercontent.com/63044364/207946801-fd42c32b-60ef-4a35-bdfe-5dbb86a8d6de.png)

And I have googled and read articles, but all of them talk about only two pages, `client.html and server.html`, whereas mine is three, `client.html, nodejs.html, and edge.html`. 

Has the feature been updated, with the documentation not reflecting the new changes, or is this an issue that is applicable to only me?

**Comment by cuongnq610** [comment]
As you know, Nextjs uses Edge to implement middlewares because it provides sufficient features to tasks efficiently, making it faster.

For this reason, we have an `edge.html` file in the output, which display analyzed data for the middleware.

**Reply by Timonwa** [reply]
Thank you.@cuongnq610
